src/components/RichText/_rich-text.scss

```diff
@@ -7,8 +7,12 @@
 // ----------------------------------
 
 .cc-rich-text {
+
```

Comment:
Was this space added by the linter? Not sure it should be there, but potentially that's a linting issue 😬

Reply:
yes, it got linted! Not my choice

Reply:
dang linter